server.go

```diff
@@ -674,19 +674,18 @@ func (r *oauthProxy) newOpenIDClient() (*oidc.Client, oidc.ProviderConfig, *http
 			ID:     r.config.ClientID,
 			Secret: r.config.ClientSecret,
 		},
-		HTTPClient:        hc,
```

Comment:
so the client is no longer used by the underlining lib? .. as the probably break the https://github.com/gambol99/keycloak-proxy/blob/master/server.go#L628-L647 .. I recall this was added to enable skip-tls, custom proxy setting etc ..

Reply:
@gambol99 as far as I can tell, nothing was broken after the removal. But we can run some tests before take the decision of merging it.